files/build_templates/docker_image_ctl.j2

```diff
@@ -236,6 +257,11 @@ start() {
         --tmpfs /tmp \
 {%- endif %}
         --tmpfs /var/tmp \
+{%- if docker_container_name == "database" %}
+        --env "NAMESPACE_ID"="$DEV" \
+        --env "NAMESPACE_PREFIX"="$NAMESPACE_PREFIX" \
+        --env "NAMESPACE_COUNT"=$NUM_ASIC \
```

Comment:
>NAMESPACE_COUNT [](start = 15, length = 15)

Why need the count?

Reply:
The Namespace count is passed internally as an environment variable when we create database docker. It is used in the script database_global.json.j2 to generate the actual file database_global.json dynamically based on the number of ASIC's.